src/components/Form Fields/field-input.tsx

```diff
@@ -30,17 +32,27 @@ export default function FieldInput<T extends FieldValues = FieldValues>({
   label,
   placeholder,
   icon,
+  type,
+  width = "full",
   ...props
 }: FormProps<T>) {
   const { field, fieldState } = useController(props);
-  const errorStyle = !fieldState.invalid
-    ? "relative mb-2 flex h-10 w-full flex-row items-center justify-between rounded-lg border border-grey-300 px-3 shadow"
-    : "relative mb-2 flex h-10 w-full flex-row items-center justify-between rounded-lg border border-red-500 px-3 shadow";
+
+  const baseStyle = `relative mb-2 flex h-10 w-${width} flex-row items-center justify-between rounded-lg border px-3 shadow hover:shadow-grey-300`;
```

Comment:
The dynamic class `w-${width}` will not work for the client because of how tailwindcss works. Please define the entire class

Reply:
Ah ok, should be fixed now